kubectl-fdb/cmd/k8s_client_test.go

```diff
@@ -151,7 +151,7 @@ var _ = Describe("[plugin] using the Kubernetes client", func() {
 
 		DescribeTable("should show all deprecations",
 			func(tc testCase) {
-				pods, err := getAllPodsFromClusterWithCondition(k8sClient, clusterName, namespace, tc.conditions)
+				pods, _, err := getAllPodsFromClusterWithCondition(k8sClient, clusterName, namespace, tc.conditions)
```

Comment:
Can we build a test case where one Pod is missing and we check the output of stderr? We can use `bytes.Buffer` for the stderr `io.Writer` and check the content later, here is an example: https://github.com/FoundationDB/fdb-kubernetes-operator/blob/main/kubectl-fdb/cmd/version_test.go#L44-L49.

Reply:
I am not sure how to run kubectl-fdb tests. Running make test does not execute them
```
?       github.com/FoundationDB/fdb-kubernetes-operator/kubectl-fdb     [no test files]
```

So I wrote the test adHoc.

Reply:
This is not an e2e test, everything is mocked. The plugin has no e2e tests.

Reply:
Got the error locally. Sorry for the noise.

Reply:
I initially wrote a passing test and looking at 
```
?       github.com/FoundationDB/fdb-kubernetes-operator/kubectl-fdb     [no test files]
```
I got confused. It has been a long day 😓